docs/scheduled-transactions/revised-spec.md

```diff
@@ -0,0 +1,157 @@
+transaction has already been executed.
+
+### Paying for scheduled transactions
+
+If the `ScheduleCreate` gives a `scheduledPayerAccountID`, the network will charge this payer the service fee 
+for the scheduled transaction at the consensus time that it executes (if ever). If no such payer is specified, 
+the network will charge the payer of the originating `ScheduleCreate`.
+
+### Receipts and duplicate creations
+
+When a `ScheduleCreate` resolves to `SUCCESS`, its receipt includes a new `ScheduleID` field
+with the id of the created schedule. Its receipt _also_ includes a new 
+`scheduledTransactionID` field with the `TransactionID` that can be used to query for the 
+record of the scheduled transaction's execution. 
+
+This `scheduledTransactionID` will be the `TransactionID` of the `ScheduleCreate`, 
+with a new additional field `scheduled=true`.  However, clients should always use the id 
+from the receipt instead of relying on this correspondence.
+
+#### Duplicate creations
+There is a special case in which a `ScheduleCreate` transaction tries to re-create a
+schedule that already exists in state. When this happens, the `ScheduleCreate` resolves 
+to a new status of `IDENTICAL_SCHEDULE_ALREADY_CREATED`, and the `ScheduleID` in its receipt points
+to the existing schedule. A client receiving `IDENTICAL_SCHEDULE_ALREADY_CREATED` can
+then submit a `ScheduleSign` (see below) with the given `ScheduleID`, signing with
+the same Ed25519 keys it used for its own create attempt. 
+
+Note the `scheduledPayerAccountID` field does not affect the identity of a schedule;
+that is, if two `ScheduleCreateTransactionBody` instances agree in their first four
+fields, they are considered identical.
+  
+### Enforced checks
+
+The only body-specific precheck enforced for a `ScheduleCreate` transaction is that the 
+`memo` field is valid. At consensus, the following checks are enforced:
+  1. The `memo` must be valid.
+  2. The type of the `scheduledTransactionBody` must be in the `scheduling.whitelist`; and it 
+     must reference only non-deleted entities that exist in state at the time the `ScheduleCreate` 
+     reaches consensus; and it must have an empty `TransactionID`. Its `nodeAccountID` 
+     may be left empty.
+  3. The `adminKey` must be valid, if present.
+  4. The `executeImmediatelyOnValidSignatures` field must be `true`.
+  
+## The `ScheduleSign` transaction
+  
+The `ScheduleSignTransactionBody` message is to be,
+  
+```  
+message ScheduleSignTransactionBody {  
+  ScheduleID scheduleID = 1; // The id of an existing schedule to affirm signing keys for
+}  
+```  
+
+Suppose we use the `scheduleID` field to point to schedule `0.0.12345`, which contains a 
+scheduled transaction `X`. If any of the Ed25519 keys required to sign `X` sign 
+our `ScheduleSign` ordinary transaction, then the network will update the state of 
+schedule `0.0.12345` with these keys.
```

Comment:
This doc leaves out some info from the other spec like when and how a transaction is executed when enough signatures are collected. Are we intending to flesh it out when the two are merged later?

Reply:
I added some comments in the `ScheduleCreate` proto, but I'm not sure we have a long-term contract we want to advertise. Will think about it.

Reply:
I mean, it's kind of the whole point of scheduled transactions so think it's important to mention in this doc. Something about how the transaction will immediately execute after enough signatures have signed either the schedule create or sign transactions. Also about which signatures need are necessary to sign (not concrete as it varies per inner transaction type).

Reply:
Fair enough! 😁  Added two sections titled "Ready-to-execute status" and "Triggered execution" right after the introduction.